### DISCUSSION TITLE
unload images

### DISCUSSION THREAD

**Discussion by singersbalm** [emilk/egui#3944] [Q&A] [answered]
how do I unload images?. I can create an image with ui.
add(
    egui::Image::new(egui::include_image!("../../assets/ferris.png"))
        .rounding(5.0)
);

or I create an image with [from_bytes](https://docs.rs/egui/latest/egui/widgets/struct.Image.html#method.from_bytes)

and after that I can access it with the uri. But what if it's only a temporary image I want to dispose of? how do I get rid of it?

**Comment by YgorSouza** [comment, marked as answer]
There's the method [Context::forget_image](https://docs.rs/egui/0.25.0/egui/struct.Context.html#method.forget_image).
